framework/include/base/FEProblemBase.h

```diff
@@ -665,6 +666,13 @@ class FEProblemBase :
   MooseSharedPointer<MultiApp> getMultiApp(const std::string & multi_app_name);
 
   /**
+   * Execute MultiAppTransfers associate with execution flag and direction.
+   * @param type The execution flag to execute.
+   * @param direction The direction (to or from) to transfer.
+   */
+  void execMultiAppTransfers(ExecFlagType type, MultiAppTransfer::DIRECTION direction);
```

Comment:
With respect to "DIMENSION", all I did is give it a name so that I can use it as a function input.